bindings/go/unicorn/uc.h

```diff
@@ -1,2 +1,3 @@
 uc_err uc_reg_read_batch_helper(uc_engine *handle, int *regs, uint64_t *val_out, int count);
 uc_err uc_reg_write_batch_helper(uc_engine *handle, int *regs, uint64_t *val_in, int count);
+void uc_ctl_set_cpu_model_helper(uc_engine *handle, int model);
```

Comment:
This is also wrong, the type of the return value is uc_err.

Reply:
Fixed